### DISCUSSION TITLE
snowflake connectivity  Update Table Schema or Sourcing new tables -- failure

### DISCUSSION THREAD

**Discussion by cosminioan-sonos** [TabularEditor/TabularEditor3#990] [Other Topics]
hi Daniel,
still having problems with the ODBC connectivity within TE3, but ONLY when I attempt to do an "Update Table Schema" OR  when attempting  to **add new tables** off of the same ODBC connection , I get the following error:
![image](https://github.com/TabularEditor/TabularEditor3/assets/121896036/8b5e13d2-5fe7-4652-9f87-051cac7e4ec7)

I have NO problem REFRESHING the entire db or individual tables (within TE or SSMS), once deployed  -- it all works fine when trying to process.

I feel there's something going on with the M methods or code, somewhere, it's not carrying  through *all the connection properties*, that otherwise seem fine when processing the db/tables.  Doing a subsequent table import or  table metadata refresh works just fine in Visual Studio. (the TE is using the bim file created by VS)

(This seems to be what I had encountered, in issue #973,   before, but I was not able to pinpoint the problem with more accuracy.)

any thoughts?
thanks kindly --
Cos

**Comment by mlonsk** [comment]
Hi @cosminioan-sonos 
We are treating this as a bug and have opened issue #993 to track it. 
We are looking at getting it fixed.

**Reply by cosminioan-sonos** [reply]
great, thank you for the acknowledgement.  I have sent a separate email to support just the other day, with repro details and videos.
thanks again,
Cosmin

**Reply by otykier** [reply]
Fixed in [3.12.0](https://github.com/TabularEditor/TabularEditor3/releases/tag/3.12.0), thanks for reporting!
